src/steps/containers.rs

```diff
@@ -48,7 +48,7 @@ impl Display for Container {
 
 /// Returns a Vector of all containers, with Strings in the format
 /// "REGISTRY/[PATH/]CONTAINER_NAME:TAG"

```

Comment:
Might be better to update this doc comment as well:

```suggestion
/// Returns a Vector of all containers, with Strings in the format
/// "REGISTRY/[PATH/]CONTAINER_NAME:TAG"
///
/// Containers specified in `ignored_containers` will be filtered out.
```